### ISSUE TITLE
Float to top doesn't keep imports below module docstring

### ISSUE THREAD

**Issue by ericmjl** [PyCQA/isort#1541]
I wasn't sure if this is a bug or not, but I thought I'd report it to see whether this is intended behaviour.

When I have a module written as such:

```python
"""Module-level docstring."""

def func():
    pass

import time

def func2():
    pass
```

After passing through `isort`, I get:

```python
import time

"""Module-level docstring."""

def func():
    pass

def func2():
    pass
```

However, this conflicts with `pydocstyle` error `D100` (Missing docstring in public module.) The behaviour that would not conflict with error `D100` is:

```python
"""Module-level docstring."""

import time

def func():
    pass

def func2():
    pass
```

May I ask, is the behaviour that I reported intended, or should `isort` be moving modules to the top but below the module-level docstring?

**Comment by timothycrosley** [comment]
Hi @ericmjl,

You are correct that this is a bug! It was also mentioned in #1499 and has been fixed in the 5.6.0 release of isort: https://pycqa.github.io/isort/CHANGELOG/#560-october-7-2020

Thanks!

~Timothy

**Comment by ericmjl** [comment]
Thank you, @timothycrosley!
